File: clang/071024_B4_21665.c
/*
 * https://www.acmicpc.net/problem/21665
 */
#include <stdio.h>
#include <stdlib.h>

int main( void )
{
	int i, j;
	int r, c;
	char img[ 100 ][ 100 ] = { 0 };
	char cpy[ 100 ][ 100 ] = { 0 };
	int ret;
	
	scanf( "%d %d\n", &r, &c );
	
	for ( i = 0; i < r; i++ ) {
		scanf( "%s", img[ i ] );
	}
	for ( i = 0; i < r; i++ ) {
		scanf( "%s", cpy[ i ] );
	}
	
	ret = 0;
	for ( i = 0; i < r; i++ ) {
		for ( j = 0; j < c; j++ ) {
			if ( (img[ i ])[ j ] == (cpy[ i ])[ j ] ) {
				ret++;
			}
		}
	}
	
	printf( "%d\n", ret );
	
	return EXIT_SUCCESS;
}

File: cshap/071024_B4_21665.cs
/*
 * https://www.acmicpc.net/problem/21665
 */
#nullable disable
using System;
using System.Text;

class B4_21665
{
	//static void Main()
	//{
	//	Console.WriteLine("--- Started ---");
	//	Solution();
	//	Console.WriteLine("--- Finished ---");
	//}
	
	static void Solution()
	{
		int ret = 0;
		int[] nm = Console.ReadLine().Split().Select(int.Parse).ToArray();
		string[] map = new string[nm[0]];
		string[] cpy = new string[nm[0]];
		for (int i = 0; i < nm[0]; i++) {
			map[i] = Console.ReadLine();
		}
		Console.ReadLine();
		for (int i = 0; i < nm[0]; i++) {
			cpy[i] = Console.ReadLine();
			for (int j = 0; j < nm[1]; j++) {
				if (map[i][j] == cpy[i][j]) ret++;
			}
		}
		Console.WriteLine(ret);
	}
}
